### PR TITLE
Angular hook for waiting for pending request should rely on ng-app instead of <body>

### PR DESCRIPTION
**Pull request by bartoszmajsak**

### CODE REVIEW

File: graphene/ftest/pom.xml [hunk 1]
```diff
@@ -50,6 +52,19 @@
...
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>${guava.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>${assertj-core.version}</version>
```
Comment (aslakknutsen) [inline graphene/ftest/pom.xml]: test scope?

Comment (bartoszmajsak) [inline graphene/ftest/pom.xml]: Fixed :)

**Comment by bartoszmajsak** [comment]
Any plans for the release and including this change?

**Comment by kenfinnigan** [comment]
Apologies @bartoszmajsak 

My wife has been away for the last week, so I've been pulling double duty with the kids!

I've taken a quick look now, and it seems fine. But I was curious as to where Guava was used? It was added to test scope, but couldn't see an import for it anywhere.

File: graphene/ftest/src/test/java/org/jboss/arquillian/graphene/angular/ftest/pageobjects/TodoList.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+    @FindByNg(repeat = "todo in todos")
+    private List<WebElement> todoRepeat;
+
+    @Drone
+    private WebDriver driver;
+
+    public List<TodoItem> openTasks() {
+        final List<WebElement> currentTodos = driver.findElements(ByAngular.repeat("todo in todos"));
+        return Lists.transform(currentTodos, new Function<WebElement, TodoItem>() {
```
Comment (bartoszmajsak) [inline graphene/ftest/src/test/java/org/jboss/arquillian/graphene/angular/ftest/pageobjects/TodoList.java]: Guava is used in this case. Just for easier for testing. If you feel like it's too much for this simple task, I can take it out from tests.

**Comment by kenfinnigan** [comment]
Ah, missed the import before. It's not a problem

**Pull request merged**